plugins/rdsr-ps/fare/Makefile

```diff
@@ -0,0 +1,40 @@
+# Set INSTALLDIR environment variable to the directory containing
+# the IRATI installation
+
+INSTALLDIR=/users/krausch/pristine/userspace
```

Comment:
Remove this line, INSTALLDIR has to be set by the caller, and must not have hardwired paths